docs/docusaurus/sidebars.json

```diff
@@ -181,6 +182,14 @@
       "contributing/contribute_codeowners"
     ],
     "Technical Reference": [
+      {
+        "type": "subcategory",
+        "label": "General",
+        "ids": [
+          "resources/pcap",
```

Comment:
For both ids: prefix them both with either `dev_` or `ref_` for namespacing purposes